bot/cogs/source.py

```diff
@@ -16,15 +16,23 @@ class Source(commands.Cog):
     def __init__(self, bot: commands.Bot) -> None:
         self.bot = bot
 
+    @in_channel(constants.Channels.bot)
     @commands.command(name="source")
-    async def command_source(self, ctx: commands.Context, command_name: str = None) -> None:
+    async def command_source(self, ctx: commands.Context, command_name: str = None, sub_command: str = None) -> None:
         """View the source of a command."""
         if command_name is None:
-            return await ctx.send("> https://github.com/python-discord/bot")
+            await ctx.send("> https://github.com/python-discord/bot")
```

Comment:
Not sure why wouldn't we do `return await ctx.send(...)` in one line honestly

Reply:
If we're annotating that our function returns `None` then it should return `None`, not maybe `None`

Reply:
Okay, I have just checked in discord.py. `send` returns a `discord.Message` instance, well then, we should return `None`.